setup.sh

```diff
@@ -0,0 +1,63 @@
+set -e
```

Comment:
@bencalegari I ran this and everything worked successfully! `brew update` took a long time, but I haven't ran it in awhile and that's somewhat expected.

Do you think we'd have another setup script in the starter app or would we point to this script in the starter app?

Reply:
The latter seems sufficient for now. Glad it worked!